spec/druid_minter_spec.rb

```diff
@@ -1,18 +1,14 @@
 require 'spec_helper'
 
 describe PreAssembly::DruidMinter do
-  before(:all) do
-    @minter = described_class
+  it "should get a sequence of druids from calls to next()" do
```

Comment:
the test titles here seem to go against the preferred style of not using "should" in the test title and just describing what happens in the scenario where the tests pass.  just pointing that out in case you care.  that was never a bit of style that i cared about one way or the other, so i certainly wouldn't block a merge over it.

Reply:
ah, that's just a rebase issue.  the commit for that happened in parallel w/ this overhaul

Reply:
I'd punt on it for now and follow up w/ more rubocop autocorrections later (which is all the other commit did).

Reply:
OK, rather than introduce rebase issues, I fixed it in the follow up PR #112